### DISCUSSION TITLE
Roadmap for D365 Patterns and Practice Project

### DISCUSSION THREAD

**Discussion by swidz** [microsoft/dynamics365patternspractices#12] [General]
Dear Eva and Rachel,

First off all, Thank you for a wonderful work you are doing. The standardised process and pattern list is something that our community was waiting for a long time! 

We have already started to utilise documents you are preparing and are very curious about the roadmap of the d365 patterns & practices project (D365P&P). 

Our plan is to extend your work for our d365 based solutions and our d365 implementation methodology, thus we are interested knowing which outcomes of the D365P&P are likely being frequently updated and which we may consider completed.
How often do you plan do update d365 process list, do you consider the existing list a complete catalogue and only new process will be added along new functionality released or it is still a draft that will evolve.
It seems at the moment the most important aspect is knowing the road map regarding the business process catalogue but also other related questions arise:

- Can you share your plans on what types of documents / templates will be delivered,
- How detailed the documents will be e.g. what level of process description we can expect in future,
- Can we expect more document templates for each part of blueprint ,
- Can we expect process maps for all business areas covered by d365 (production, order to cash, inventory, plan to delivery, etc.),
- What detail level of process diagrams we can expect.

Best regards,
Sebastian Widz

**Comment by edupont04** [comment]
Hello Sebastian,
Thank you for your feedback! We're delighted to hear that you find our content useful.

The guidance content for Microsoft Dynamics 365 is an ongoing project with contributors from across Microsoft. Specifically the business processes and related patterns is currently based on the catalog in the Excel file at https://github.com/microsoft/dynamics365patternspractices/tree/main/templates/business-processes

The catalog outlines content that we believe captures each end-to-end process at an increasingly detailed level. Learn more at https://learn.microsoft.com/en-us/dynamics365/get-started/contribute#business-process-contributions

Currently, we at Microsoft have identified 13 end-to-end scenarios, but more may be added later. You can suggest a new end-to-end scenario by submitting an Issue of type _Business process catalog change request_ here: https://github.com/microsoft/dynamics365patternspractices/issues/new/choose  

We _very much_ welcome contributions from the community. We don't have a fixed roadmap due to resource constraints, but the Excel file contains our vision. Keep the feedback coming!
